obs-ndi.spec

```diff
@@ -0,0 +1,107 @@
+#*********************************************************************************************************
+#*   __     __               __     ______                __   __                      _______ _______   *
+#*  |  |--.|  |.---.-..----.|  |--.|   __ \.---.-..-----.|  |_|  |--..-----..----.    |       |     __|  *
+#*  |  _  ||  ||  _  ||  __||    < |    __/|  _  ||     ||   _|     ||  -__||   _|    |   -   |__     |  *
+#*  |_____||__||___._||____||__|__||___|   |___._||__|__||____|__|__||_____||__|      |_______|_______|  *
+#* http://www.blackpantheros.eu | http://www.blackpanther.hu - kbarcza[]blackpanther.hu * Charles Barcza *
+#*************************************************************************************(c)2002-2019********
+# 	Soec written by Charles K Barcza for blackPanther OS - www.blackpanther.hu
+
+%define genlibndi 1
+
+Name: 	 obs-ndi
+Version: 4.6.0
+Release: %mkrel 2
+Source0: %name-%version.tar.gz
+Source1: obs-studio-23.1.0-rc1.tar.gz
+Patch0:	 %name-libdir.patch
+Summary: OBS broadcast to OBS
+License: GPL
+Group:   Multimedia/Video
+Requires: %{_lib}ndi3 >= 3.8.0
```

Comment:
is there an RPM spec providing this?

Reply:
RPM generated with two different license originally, if user want can build from source, or use your deb package as well for generate a RPM package with non-free license. But you right! The "requires" my fold, I've fixed to as recommends optional package and fixed the license as well on my git